data/defaults.yaml

```diff
@@ -227,19 +227,12 @@ prometheus::statsd_exporter::download_extension: 'tar.gz'
 prometheus::statsd_exporter::download_url_base: 'https://github.com/prometheus/statsd_exporter/releases'
 prometheus::statsd_exporter::extra_groups: []
 prometheus::statsd_exporter::group: 'statsd-exporter'
-prometheus::statsd_exporter::mapping_config_path: '/etc/statsd_mappings.conf'
-prometheus::statsd_exporter::maps:
-  - 'map': 'test.dispatcher.*.*.*'
-    'name': 'dispatcher_events_total'
-    'labels':
-      'processor': '$prometheus::1'
-      'action': '$prometheus::2'
-      'outcome': '$prometheus::3'
-      'job': 'test_dispatcher'
+prometheus::statsd_exporter::mapping_config_path: '/etc/statsd-exporter-mapping.yaml'
+prometheus::statsd_exporter::mappings: []
 prometheus::statsd_exporter::package_ensure: 'latest'
 prometheus::statsd_exporter::package_name: 'statsd_exporter'
 prometheus::statsd_exporter::user: 'statsd-exporter'
-prometheus::statsd_exporter::version: '0.3.0'
+prometheus::statsd_exporter::version: '0.8.0'
```

Comment:
Can you please keep the old version? Otherwise it would be a breaking change. We want to do that in a later release for all exporters.

Reply:
The style issues are fixed now and saw that i didn't see that versions older than v0.7.0 still used the single dash options so i added some logic for that too.

Reply:
I also did an experiment to see how difficult it would be to support older versions as well but it gets quite funky and i think more confusing with the different file format and names..

Reply:
We did a release of this module a few days ago. In the current merge window we want to update all exporters, so this is fine now.